src/api.cc

```diff
@@ -173,3 +173,19 @@ bool AkliteClient::IsRollback(const TufTarget& t) const {
 
   return known_local_target(*client_, target, known_but_not_installed_versions);
 }
+
+InstallResult AkliteClient::SetSecondaries(const std::map<std::string, SecondaryEcu>& ecus) {
+  Json::Value data;
```

Comment:
`SecondaryEcu::hwid` is not used at all?

Reply:
used in GetLatest, but we don't need it on on the backend.